src/style.css

```diff
@@ -1 +1,64 @@
 /* Styles go here */
+* {
+  margin: 0;
+}
```

Comment:
point 9 from [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)

you need to reset margins for tags